trunk/modules/python/dataset/ingest/labeled.py

```diff
@@ -228,7 +240,13 @@ def hdf5Dataset(filepath, holdoutPercentage=.05, minTest=5,
     # Here we create the handles to the data buffers. This operates on the
     # assumption the dataset may not fit entirely in memory. The handles allow
     # data to overflow and ultimately be stored entirely on disk. 
-    imageShape = list(getImageDims(train[0][0], log))
+
+    # find the most common file size and use to get the image
+    # shape
+    commonSize = mostCommon(train, lambda t: os.path.getsize(t[0]))
+    exampleImage = (t for t in train
+                    if os.path.getsize(t[0]) == commonSize).next()
+    imageShape = list(getImageDims(exampleImage[0][0], log))

```

Comment:
This was throwing an error when I used it. If you change the --holdout or --batch options in the application level it should prompt getting to this piece of code to see the error.